integrations/venice-beam/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+  id 'io.github.lhotari.gradle-nar-plugin'
+}
+
+dependencies {
+  implementation(project(':internal:venice-common')) {
+    exclude module: 'kafka_2.10'
+    exclude group: 'org.scala-lang'
+  }
+  implementation project(':clients:da-vinci-client')
+  implementation project(':clients:venice-thin-client')
+  implementation project(':clients:venice-client')
+
+  implementation libraries.log4j2api
+  implementation libraries.log4j2core
+
+  implementation libraries.beamSdk
+  implementation libraries.beamExtensionAvro
+}
+
+ext {
+  // to be tested in integration test
+  jacocoCoverageThreshold = 0.00
```

Comment:
This is not great but for the initial commit we could let it go... I assume there are no tests in the internal project where this is coming from? Or are there any we may be able to contribute here as well?

In any case, it would be good to bump it up in the future.

Reply:
The tests are internal application specific and unfortunately no generic tests that can be ported currently. We relied on testing in dark environment given the dependency on Venice and Beam. Yes agree on tests should be added in future.